microsoft/utils/aitl/tier0-sig.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
tier0-sig and tier0-vhd are enough. Because those templates cannot be used directly, so it doesn't worth to provide multiple files for convenient.

Reply:
Makes sense, will get rid of others and just explain how to use the different tiers in docs